flutter/Graminsta/lib/post/followUser.dart

```diff
@@ -0,0 +1,102 @@
+import 'dart:async';
+import 'dart:convert';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:Graminsta/service/http_service.dart';
+import 'package:Graminsta/service/follow_service.dart';
+import 'package:Graminsta/models/user.dart';
+
+Future<List<User>> fetchUsers() async{
```

Comment:
Comma after `async`

Reply:
I looked up the dart docs and maybe you mean space rather than comma?

Reply:
Oh, right, space.